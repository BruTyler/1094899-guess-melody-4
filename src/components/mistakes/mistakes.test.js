import React from 'react';
import renderer from 'react-test-renderer';
import Mistakes from './mistakes.jsx';

describe(`Mistakes render suit`, () => {
  it(`Mistakes render case`, () => {
    const generatedTree = renderer.create(
        <Mistakes count={3}/>
    ).toJSON();

    expect(generatedTree).toMatchSnapshot();
  });
});
