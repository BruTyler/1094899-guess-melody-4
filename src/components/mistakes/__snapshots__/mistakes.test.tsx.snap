// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mistakes render suit Mistakes render case 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
  <div
    className="wrong"
  />
</div>
`;
